client/src/Game.h

```diff
@@ -12,14 +12,18 @@
 #include "NetworkThread.h"
 #include <Tilemap.h>
 #include <QuadTree.h>
+#include <EntityManager.h>
 
 
 class Game {
     sf::RenderWindow mWindow;
-    Player player;
-    PlayerCommands playerCommands;
+
+    EntityManager *entityManager;
```

Comment:
Let's be consistent !
We must decide one unique way, otherwise it will quickly turn into a mess.